docker/rebuild4.sh

```diff
@@ -32,6 +40,6 @@ mkdir -p ./docker/bind_mounts/control/mnt/vectorized/entrypoint
 mkdir -p ./docker/bind_mounts/client1/mnt/vectorized/workloads/logs
 mkdir -p ./docker/bind_mounts/client1/mnt/vectorized/entrypoint
 
-chmod a+rw -R ./docker/bind_mounts
+chmod -R a+rw ./docker/bind_mounts
```

Comment:
On MacOS, `-R` needs to come first.  So let's do it the way that works on both MacOS and Linux.